### PR TITLE
Moved mentorship_guidelines.md to content folder

### PR DESCRIPTION
**Pull request by Chinonyemildred**
I just moved mentorship_guidelines.md file to content folder and created a vue page for it.

### CODE REVIEW

File: content/mentorship_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+
+# Galaxy Mentorship Network Guidelines
+
```
Comment (bebatut) [inline content/mentorship_guidelines.md:3]: Can you put the title in the metadata and also add the id, i.e. the page name?
Sth like

```
---
id: about
title: About the Galaxy Mentorshing Network Program
---
```

File: src/pages/Mentorship_guidelines.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            path
+        }
+    }
+}
+</page-query>
+
+<script>
+export default {
+    metaInfo: {
+        title: "GMN - Mentorship_guidelines"
```
Comment (bebatut) [inline src/pages/Mentorship_guidelines.vue:24]:
```suggestion
        title: "GMN - Mentorship Guidelines"
```

File: content/mentorship_guidelines.md (hunk 1 above)
Comment (Chinonyemildred) [reply content/mentorship_guidelines.md:3]: Okay, I'll do that now

Comment (Chinonyemildred) [reply content/mentorship_guidelines.md:3]: I think the title might be Mentorship Guidelines since it's not the 'About page'. What do you think?

Comment (bebatut) [reply content/mentorship_guidelines.md:3]: Yes. What I pasted was the example from the about page

Comment (Chinonyemildred) [reply content/mentorship_guidelines.md:3]: Okay, on it now :)

**Pull request closed without merging**